source/MRViewer/MRUISaveChangesPopup.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <string>
+#include <functional>
+
+#include "exports.h"
+
+namespace MR
+{
+
+namespace UI
+{
+
+struct SettingsSaveChangesPopup
+{
+    // menu scaling
+    float scaling = 1.0f;
+    // the text for the button, if you need to exit the application
+    std::string out = "Sign out";
+    // text for button if need save scene
+    std::string dontSave = "Don't Save";
```

Comment:
comment does not fit to the parameter
```suggestion
    // text that is shown if we have changes but don't want to save them
    std::string dontSaveText = "Don't Save";